ansible/roles_ocp_workloads/ocp4_workload_kubevirt/files/csc.cr.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: operators.coreos.com/v1
```

Comment:
Do we still need CatalogSourceConfigs? I was under the impression these are obsolete these days...